### PR TITLE
Issue 25

### PR DESCRIPTION
**Pull request by jose-oliveira**
See issue #25 for discussion.

### CODE REVIEW

**Comment by nerdstein** [comment]
Needs rebased from recent merges

**Comment by nerdstein** [comment]
Naming conventions look off: securepages_baseurl as a function name in the service.

Can you clean up?

File: src/EventSubscriber/SecurePagesEventSubscriber.php [hunk 1]
```diff
@@ -52,15 +44,22 @@ public function checkForHttp(GetResponseEvent $event) {
 
       $securepagesservice = \Drupal::service('securepages.securepagesservice');
       $redirect = $securepagesservice->securePagesRedirect();
+      $securepages_baseurl = $securepagesservice->securepages_baseurl($redirect);
```
Comment (nerdstein) [inline src/EventSubscriber/SecurePagesEventSubscriber.php]: Would like this function name cleaned up for consistency

**Comment by jose-oliveira** [comment]
Rebased and cleaned-up done.

**Pull request merged**